### PR TITLE
Modified gridster to be responsive and cleaned up third party libraries

### PR DESCRIPTION
**Pull request by lasanthaS**

### CODE REVIEW

File: apps/portal/libs/gridster_0.5.6-0/jquery.gridster.js [hunk 1]
```diff
@@ -3637,6 +3636,7 @@
     */
     fn.generate_stylesheet = function(opts) {
         var styles = '';
+        var lgStyles = '';
```
Comment (TanyaM) [inline apps/portal/libs/gridster_0.5.6-0/jquery.gridster.js]: make variable name meaningful

File: apps/portal/theme/templates/designer.jag [hunk 2]
```diff
@@ -150,6 +150,15 @@
                         <div class="sidebar-brand">Block size in grid</div>
                         <div class="add-padding-3x">
 
+                            <div class="form-inline clearfix">
+                                <label class="control-label">Show Grid</label>
```
Comment (TanyaM) [inline apps/portal/theme/templates/designer.jag]: Localization

File: apps/portal/libs/gridster_0.5.6-0/jquery.gridster.js (hunk 1 above)
Comment (lasanthaS) [inline apps/portal/libs/gridster_0.5.6-0/jquery.gridster.js]: This file is no longer available

File: apps/portal/theme/templates/designer.jag (hunk 2 above)
Comment (lasanthaS) [inline apps/portal/theme/templates/designer.jag]: This is fixed

**Pull request merged**